--- nerdtree_plugin/git_status.vim
+++ nerdtree_plugin/git_status.vim
@@ -79,7 +79,6 @@ let s:need_migrate_vals = {
             \ 'g:NERDTreeMapNextHunk':        'g:NERDTreeGitStatusMapNextHunk',
             \ 'g:NERDTreeMapPrevHunk':        'g:NERDTreeGitStatusMapPrevHunk',
             \ 'g:NERDTreeShowIgnoredStatus':  'g:NERDTreeGitStatusShowIgnored',
-            \ 'g:NERDTreeIndicatorMapCustom': 'g:NERDTreeGitStatusIndicatorMapCustom',
             \ }

