diff --git a/~/.vim/pack/plugins/start/vim_codex/python/plugin.py b/plugin.py
index ae385a3..2d40b9e 100644
--- a/~/.vim/pack/plugins/start/vim_codex/python/plugin.py
+++ b/plugin.py
@@ -54,7 +54,7 @@ def initialize_openai_api():
 
 def complete_input_max_length(input_prompt, max_input_length=MAX_SUPPORTED_INPUT_LENGTH, stop=None, max_tokens=64):
     input_prompt = input_prompt[-max_input_length:]
-    response = openai.Completion.create(engine='code-davinci-001', prompt=input_prompt, best_of=1, temperature=0.5, max_tokens=max_tokens, stream=USE_STREAM_FEATURE, stop=stop)
+    response = openai.Completion.create(engine='text-davinci-001', prompt=input_prompt, best_of=1, temperature=0.5, max_tokens=max_tokens, stream=USE_STREAM_FEATURE, stop=stop)
     return response
 
 def complete_input(input_prompt, stop, max_tokens):
